setup.py

```diff
@@ -10,7 +10,7 @@
 
 from setuptools import setup, find_packages
 
-test_deps = ["pytest", "pytest-cov", "pytest-asyncio", "hypothesis<6"]
```

Comment:
What was the reason for using Hypothesis versions below 6? This was causing a dependency conflict.

Reply:
I can't remember, I think there was some sort of breaking change or something that broke things at some point, but I don't think it's an issue anymore. It was never an issue in this repo though, does hypothesis even get used?

Reply:
You typically should bind it to a specific version though.

Reply:
We'll fix it later...